api/Dockerfile

```diff
@@ -1,10 +1,8 @@
-FROM alpine:3.19 AS base_build
+FROM node:22-alpine AS base_build
```

Comment:
This feels like a significant change. alpine haven't yet added node 22 to their packages, but this official node alpine package works fine. What do you think?

Reply:
Oh, also, I saw that there are two 22 node alpine images, one based on alpine:3.19 and one based on alpine:3.20 https://github.com/nodejs/docker-node/tree/main/22
Which one are we getting?

Reply:
The image is a bit more bloat (like auto adding yarn) in these images: https://github.com/nodejs/docker-node/blob/main/22/alpine3.20/Dockerfile

Reply:
Yeah I couldn't figure out why Node was adding yarn. That seems crazy...

Reply:
> Which one are we getting?

According to [the tags](https://hub.docker.com/_/node), `22-alpine` is the same as `22-alpine3.20`

Reply:
> It looks like alpine releases are really rare, but if a critical security fix gets pushed, we might need it asap.

Yeah you're right. It's a crazy circular dependency - if there's a critical flaw in Node then we need a node image to get it asap, but if there's a critical flaw in something else in alpine then we need the alpine image to get it asap.

We can always switch back to alpine if there's a flaw especially if they've published node 22 package.

Reply:
agree.